release.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+RELEASE_VERSION=${1:-0.1.0}
+RELEASE_API_VERSION=${2:-0.1.0}
```

Comment:
What would happen if we don't set the API version (so we want it to remain constant as we release some other part of Kroxy)? Will the `mvn deploy` fail because the API version already exists?

Maybe we shouldn't have default values here.

Reply:
Ah crap, I was thinking that if we had already released API 0.1.0 then it would be a no op, and up until the deploy it is (well its a waste of time but harmless) but the deploy will fail... 

Yeah defaults are probably not worth having.